src/module/retry.ts

```diff
@@ -37,16 +64,63 @@ export class Retry extends Module {
    * The amount of time to wait before retrying.
    */
   public interval: number;
+    /**
+   * The mode for the retry (constant, linear, exponential or jitter)
+   */
+  public mode: RetryMode;
+  /**
+   * The factor to be applied for the retry mode
+   */
+  public factor: number;
+  /**
+   * The maximum amount of time to wait before retrying another time
+   */
+  public maxInterval: number;
+  /**
+   * Should the first retry be done directly after a failure occurred
+   */
+  public fastFirst: boolean;
+  /**
+   * The value to adjust delay randomly based on jitter retry duration
+   */
+  private _jitterAdjustment: number;
   /**
    * A filtering callback, to modify the retry behavior.
    * @returns true (retries), false (rejects), number (retries after some delay)
    */
   public onRejection: RetryCallback;
+
+  /**
+   * The value (between 0 and 1) to adjust delay randomly based on jitter retry duration
+  */
+  public get jitterAdjustment(): number {
+    return this._jitterAdjustment;
+  }
+  /**
+   * Set the value (between 0 and 1) for jitter adjustment (to adjust delay randomly).
+  */
+  public set jitterAdjustment(val: number) {
+    this._jitterAdjustment = this.adjustJitterValue(val);
+  }
+
+  private adjustJitterValue(val: number): number {
+    if (val < 0 || val > 1) {
+      return 0.1;
+    }
+    return val;
+  }
+
   // Constructor
   constructor (options?: RetryOptions) {
     super(options);
-    this.attempts = (options?.attempts !== undefined) ? options.attempts : 2;
-    this.interval = (options?.interval !== undefined) ? options.interval : 0;
+    this.attempts = options?.attempts ?? 2;
```

Comment:
Discovered `??` a while ago, great use 👍

Reply:
Yes, this simplifies the code